homeassistant/components/hydrawise/binary_sensor.py

```diff
@@ -17,21 +18,37 @@
 from .coordinator import HydrawiseDataUpdateCoordinator
 from .entity import HydrawiseEntity
 
-BINARY_SENSOR_STATUS = BinarySensorEntityDescription(
-    key="status",
-    device_class=BinarySensorDeviceClass.CONNECTIVITY,
+CONTROLLER_BINARY_SENSORS: tuple[BinarySensorEntityDescription, ...] = (
+    BinarySensorEntityDescription(
+        key="status",
+        device_class=BinarySensorDeviceClass.CONNECTIVITY,
+    ),
 )
 
-BINARY_SENSOR_TYPES: tuple[BinarySensorEntityDescription, ...] = (
+RAIN_SENSOR_BINARY_SENSOR: tuple[BinarySensorEntityDescription, ...] = (
+    BinarySensorEntityDescription(
+        key="rain_sensor",
+        translation_key="rain_sensor",
+        icon="mdi:weather-pouring",
+        device_class=BinarySensorDeviceClass.MOISTURE,
+    ),
+)
+
+ZONE_BINARY_SENSORS: tuple[BinarySensorEntityDescription, ...] = (
     BinarySensorEntityDescription(
         key="is_watering",
         translation_key="watering",
-        device_class=BinarySensorDeviceClass.MOISTURE,
+        device_class=BinarySensorDeviceClass.RUNNING,
     ),
 )
 
 BINARY_SENSOR_KEYS: list[str] = [
```

Comment:
This isn't used for anything, so you can just remove it.

Reply:
Fixed.